distribution/src/main/resources/docker-compose/docker-compose-minimal.yml

```diff
@@ -91,6 +95,14 @@ services:
       ACTIVEMQ_URL: "nio://activemq:61616"
       CORE_AIO_URL: "http://transform-core-aio:8090"
       FILE_STORE_URL: "http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file"
+      LOGGING_LEVEL_ORG_ALFRESCO_TRANSFORM_ROUTER: TRACE
```

Comment:
Did you mean to have this in the PR?

Reply:
Yes, this logs are very helpful in problem resolving